.circleci/config.yml

```diff
@@ -159,17 +193,31 @@ jobs:
           command: |
             sudo apt-get update -y && sudo apt-get install -y postgresql-client
       - run:
-          name: Wait for vertica
+          name: Install mysql
+          command: |
+            sudo apt-get update -y && sudo apt-get install -y default-mysql-client
+      - wait_for_db:
+          db_name: Vertica
+          connect_command: vsql -h 127.0.0.1 -U dbadmin -c 'select 1;'
+      - wait_for_db:
+          db_name: MySQL
+          connect_command: echo 'select 1;' | mysql --password=hunter2 --host=127.0.0.1 -u mysqluser mysqlitest
+      # TODO: Where is the wait for postgres and wait for mysql?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker